src/server/src/node-api/proxy-api.ts

```diff
@@ -0,0 +1,21 @@
+import {TokenAPIActions} from "./token-api";
+
+export interface NodeRPCError {
+    error: string
+}
+
+export type RPCAction = TokenAPIActions | 'mnano_to_raw' | 'mnano_from_raw' | 'process' | 'work_generate' | 'price'
+
+export interface NanoRPCRequest {
```

Comment:
Could we change the name from NanoRPCRequest to TokenRPCRequest? Since this is very specific to tokens and not Nano.

Reply:
Was pondering about that name, and think `ProxyRPCRequest` is the best fitting. This type covers all input parameters to the get/post endpoints, and that might be a token request, price request or some request for the node itself.